app/controllers/git_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'json'
+require 'git'
+require 'fileutils'
+
+class GitController < ApplicationController
```

Comment:
This file should have been commited with everything else that was generated. See PR #8